### PR TITLE
zd-6613 Use adoc instead, general cleanup, and rename to CSI instead …

### PR DESCRIPTION
**Pull request by enochtsang**
* Convert to adoc, for company standardization.
* Some general cleanup.
* Renamed to CSI Statusboard from QA statusboard.

These changes are fairly subjective so let me know what you think.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+* Note that order is important for the next 2 steps.
+* Run `mongod --dbpath=./data --port 27017` in one terminal.
+* Run `node index.js` in the other terminal.
+
+. In another command line, navigate to the `client` folder.
+* Run `grunt serve`.
+
+. The CSI Statusboard will now be served at `localhost:8000`.
+
+Alternatively, execute `run.bat`.
```
Comment (henrytran28) [inline README.adoc]: I would put this line before line 36.

Comment (henrytran28) [inline README.adoc]: I would put this line before line 36.

**Comment by henrytran28** [comment]
Stop deleting my comments.

File: README.adoc (hunk 1 above)
Comment (sararathje) [inline README.adoc]: I know you didn't do this, but I agree with Henry. I think this should be at the top instead of at the bottom.

**Comment by henrytran28** [comment]
Stop deleting my comments.

> These changes are fairly subjective so let me know what you think.

**Comment by enochtsang** [comment]
Sorry henry, been deleting out of for of habit. oops did it again.

**Pull request merged**